src/Octoshift/Services/GithubApi.cs

```diff
@@ -241,7 +266,8 @@ public virtual async Task<string> GetEnterpriseId(string enterpriseName)
             {
                 var data = await _client.PostGraphQLAsync(url, payload);
 
-                return (string)data["data"]["enterprise"]["id"];
+                var entireprise_id = (string)data["data"]["enterprise"]["id"];
+                return entireprise_id;
```

Comment:
Could you revert this as I guess it was created for debugging.

Reply:
🧹 [a842577](https://github.com/github/gh-gei/pull/1276/commits/a8425779edd122246aebe73697c415819a438f33)